### DISCUSSION TITLE
Very basic questions

### DISCUSSION THREAD

**Discussion by BenjaminSaylor** [pit-ray/win-vind#132] [General]
So I am new to Vim navigation having installed Vimium in my browser. To say it has been life changing has been an understatement. 

I downloaded win-vind to try and use this on my desktop/word/etc. I used the portable app and have it running and figured out how to switch layers but how do I get the hinting system to work?! I go to GUI Normal and hit f and nothing happens? The hints and rapid pointing do not seem to work. I'm sure I'm missing something obvious when I'm not a programmer.

**Comment by pit-ray** [comment]
Thank for using it.
Perhaps you may misunderstand the `FF` as `ff`. The `FF` is the same as `<shift-f><shift-f>`.
Please refer to [the usage document](https://pit-ray.github.io/win-vind/usage/#2-gui-operation-and-window-operation).

**Comment by azinsharaf** [comment]
hi @pit-ray Is it possible to map `<shift-f><shift-f>` to `f` or even `ff`?

**Reply by pit-ray** [reply]
Yes, but there are some solution.

1. Internal mapping that effects for only win-vind
```vimrc
map <shift-f><shift-f> ff
```

2. Keystroke generation for `ff` using macros, which don't absorb the key message of `<shift-f><shift-f>`.
```vimrc
map <shift-f><shift-f> {f}{f}
```

The combined keyset cannot be mapped to the any key at low-level yet.

**Reply by azinsharaf** [reply]
it seems `ff` jumps the cursor to the middle of the screen. how can i unmap the default?
```
unmap ff
map <shift-f><shift-f> ff
```

**Reply by pit-ray** [reply]
Sorry for my late reply.
The characters in .vindrc are case-sensitive, so to unmap `FF`, which means `<shift-f><shift-f>`, you must use `unmap FF`.
